apps/gro/controllers/confirm.js

```diff
@@ -5,6 +5,7 @@ var controllers = require('hof').controllers;
 var BaseController = controllers.base;
 var path = require('path');
 var i18n = require('hof').i18n;
+var _ = require('underscore');
```

Comment:
We should totes be using lodash. I'd like us to discuss this at our next "meetup" or rather just say it's time to ditch underscore.